tests/realtime/utils.js

```diff
@@ -185,8 +192,21 @@ module.exports = {
     return JSON.parse(JSON.stringify(standardBotAction));
   },
 
+  getStandardEvent() {
+    return JSON.parse(JSON.stringify(standardEvent));
+  },
+
   forceDelete(done) {
-    tu.forceDelete(tu.db.Perspective, testStartTime)
+    tu.db.Event.destroy({
```

Comment:
curious why calling destroy directly rather than using tu.forceDelete?

Reply:
tu.forcedelete uses name and the event doesn't have so need to use a destroy